users/models.py

```diff
@@ -48,18 +48,32 @@ class CampusAmbassadorProfile(models.Model):
     college = models.CharField(max_length=128, null=False, blank=False)
 
```

Comment:
pull from master, chnages in this file are not required

Reply:
The way you have edited this file it has indentation and formatting issues. Retain this file.